src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background-color: $white;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 1.5;
+    color: $primary-text-color;
+  }
+
+  &__subtitle {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary-text-color;
+  }
+
+  &__rating-block {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews{
+    font-size: 10px;
+    line-height: 1.4;
+    color: $primary-text-color;
+  }
+
+  &__price-block{
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__price-label{
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 1.5;
+    color: $secondary-text-color;
+  }
+
+  &__price-value{
+    font-size: 16px;
+    font-weight: 700;
+    color: $primary-text-color;
+  }
+
+  &__button{
+    margin-top: 15px;
+    height: 40px;
+    width: 100%;
+    border: 0;
+    border-radius: 5px;
+    background-color: $blue;
+    color: $white;
+    font-weight: 700;
+    transition: all .1s ease-out;
```

Comment:
it's not a good practice to use `all` value as transition-property. Specify what exactly you'd like to change on hover